Fhi.HelseId.Blazor/WebApplicationExtensions.cs

```diff
@@ -65,7 +65,6 @@ public static WebApplication UseHelseIdForBlazor(this WebApplication app)
             if (options.UseCorrelationId)
             {
                 app.UseMiddleware<CorrelationIdMiddleware>();
-                app.UseHeaderPropagation();
```

Comment:
What is the reason for this being removed? Is this default behavior?

Reply:
WebApi HeaderPropagation is supposed to propagate headers from HttpContext to HttpClient.

CorrelationIdHandler from our code already does the check for correlation id in the context and adds it to the client, so we don't need to do it twice.